Gek62/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+def data
+  @data = File.read('data', encoding: 'UTF-8')
+end
+
+def blank_error
+  puts 'City name can not be blank!'
+end
+
+def mean(town, data)
+  return -1 if (numbers = extract_data(town, data)).nil?
+
+  numbers.sum / 12
+end
+
+def variance(town, data)
+  return -1 if (numbers = extract_data(town, data)).nil?
```

Comment:
The first line in `variance` method is the same as in `mean`.
It can be refactored so that you don't repeat yourself.